book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(person, date, self)
+    @rentals << rental
+  end
```

Comment:
Good job by creating add_rental method but  @rentals << rental is already done in Rental initializer so no need to redo it here